ui/de-lib/src/main/java/org/iplantc/de/analysis/client/presenter/AnalysesPresenterImpl.java

```diff
@@ -353,8 +351,13 @@ public void onShareSelected(List<Analysis> selected) {
 
     @Override
     public void setCurrentFilter(AnalysisFilter filter) {
-       this.currentFilter = filter;
-       loadAnalyses(currentFilter);
+        if(filter == null) {
+            currentFilter =filter;
+            return;
```

Comment:
User cannot clear the filter. He can only change it one value to other. The code programmatically clears the filter when user is drilling down a HT analysis or doing a search.